src/models/SessionReportPilotSupportingAttachment.js

```diff
@@ -0,0 +1,34 @@
+const { Model } = require('sequelize');
+const { afterDestroy } = require('./hooks/objectiveTemplateFile');
```

Comment:
I would remove this, it doesn't do anything related to this. I see that you copied it from 'sessionReportFile', it can be removed there as well. Just a mistake.

Reply:
Good catch. Used as starting template.